api/odilo.py

```diff
@@ -139,14 +531,8 @@ def _make_absolute_url(self, url):
             url = self.library_api_base_url + url
         return url
 
-    def get_patron_credential(self, patron, pin):
-        """Create an OAuth token for the given patron.
-
-        TODO: This code does nothing and should be removed.
-        """
-
-        def refresh(credential):
-            return self.get_patron_access_token(credential, patron, pin)
+    def refresh(credential):
+        return self.get_patron_access_token(credential, patron, pin)
         return self._patron_credential_lookup(patron, refresh)
```

Comment:
I'm not sure what's going on here, did you mean to delete these lines?

Reply:
Yes; see https://github.com/NYPL-Simplified/circulation/pull/1017